fixtures/corgi_playwright.py

```diff
@@ -0,0 +1,39 @@
+from playwright.sync_api import sync_playwright
```

Comment:
These tests should not be created within the cnx-automation repository. If we're writing tests they should live within the same repository as the source code [here](https://github.com/openstax/output-producer-service/blob/main/backend/app/tests/)

Reply:
as per our discussion, I will use these tests as my test-tools in cnx-automation but once all the UI tests were rewritten in playwright, I will look into moving these too into output-producer-service

Reply:
Ok, Otto and I discussed this a bit further. The overarching plan atm is to use cnx-automation to learn playwright and develop these tests that are useful for Otto. At the same time, Otto will work to replacing the selenium tests in CORGI with Playwright 1 for 1 until all the tests are done. At that point, we'll try to work more of the tests that Otto has in cnx-automation into CORGI so that we can test the entire pipeline after a deployment.